android/src/app/organicmaps/car/NavigationSession.java

```diff
@@ -110,9 +110,16 @@ public void onResume(@NonNull LifecycleOwner owner)
   }
 
   @Override
-  public void onDestroy(@NonNull LifecycleOwner owner)
+  public void onStop(@NonNull LifecycleOwner owner)
   {
     Logger.d(TAG);
+    LocationHelper.INSTANCE.onTransit(false);
```

Comment:
Does it mean that AppBackgroundTracker doesn't work properly with car? This particular usage is not a big deal, but there are other users of AppBackgroundTracker which can be broken.

Reply:
I don't know :)
That needs to be checked and tested. I think during the Navigation Screen implementation. 
Let's leave it for now.

Reply:
ok, we have a TODO item to check LocationHelper.